src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,87 @@
+package duke;
+
+import duke.task.Task;
+
+/**
+ * A class that deals with interactions in terminal.
+ */
+public class Ui {
+    private final String LINE = "______________________________________________________________";
+    private final String NAME = "duke";
+    private final String GREETINGS = "Hello, I'm " + NAME + "\n" + "What can I do for you?";
+    private final String GOODBYE = "Bye. Hope to see you again!";
+
+    public void greetings () {
+        System.out.println(LINE + "\n" + GREETINGS + "\n" + LINE);
+    }
+
+    public void goodbye () {
+        System.out.println(LINE + "\n" + GOODBYE + "\n" + LINE);
+    }
+
+    /**
+     * Show the message printed when a task is added.
+     */
+    public void showAddTask (TaskList tasks) {
+        System.out.println(LINE + "\n" + "Got it. I've added this task:"
+                + "\n     " + tasks.get(tasks.size() - 1).toString() + "\n"
+                + "Now you have " + tasks.size() + " task left \n" + LINE);
+    }
+
+    /**
+     * Show the message printed when a task is deleted.
+     */
+    public void showDeleteTask (TaskList tasks, int index) throws DukeException {
```

Comment:
Small inconsistency where showDeleteTask and (TaskList ...) have a space in between where most other methods don't. A few other occurrences of this in other methods in this class